sys/test_utils/netdev_eth_minimal/netdev_eth_minimal_internal.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2022 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     test_utils_netdev_eth_minimal
+ *
+ * @{
+ * @file
+ * @brief       Internal definitions for the netdev_eth_minimal module
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#include "net/netdev.h"
+#include "event.h"
+#include "init_dev.h"
+
+typedef struct {
+    event_t event;      /**< event to serve ISR */
+    netdev_t *dev;      /**< pointer to the device */
+} device_reg_entry_t;
+
+extern device_reg_entry_t _devices[NETDEV_ETH_MINIMAL_NUMOF];
```

Comment:
Should this include "netdev_eth_minimal.h"?

Reply:
`NETDEV_ETH_MINIMAL_NUMOF` comes from `init_dev.h`

Reply:
(my comment was mainly because of the static-check warning, didn't look much into it)